src/test/end-to-end/features/EmploymentRespondentDetails/verifyDidYouWorkForOrg.js

```diff
@@ -8,11 +8,11 @@ Scenario("Did you work for the organisation or person you're making your claim a
   await I.authenticateWithIdam();
   await doNotHaveToCompleteCard(I);
   await didYouWorkForOrganisation(I, 'No');
-}).tag('@RET-WIP');
+}).tag('@RET-BAT');
```

Comment:
Should this be the tag?

Reply:
This is not my test but I can revert the tag if it helps...